src/edu/csus/ecs/pc2/clics/API202306/CLICSOrganization.java

```diff
@@ -0,0 +1,167 @@
+// Copyright (C) 1989-2025 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import java.io.File;
+import java.util.HashMap;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.Constants;
+import edu.csus.ecs.pc2.core.StringUtilities;
+import edu.csus.ecs.pc2.core.log.Log;
+import edu.csus.ecs.pc2.core.log.StaticLog;
+import edu.csus.ecs.pc2.core.model.Account;
+import edu.csus.ecs.pc2.services.core.JSONUtilities;
+
+/**
+ * CLICS Organization
+ * Contains information about an Organization (institution)
+ *
+ * @author John Buck
+ *
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CLICSOrganization {
+
+    @JsonProperty
+    private String id;
+
+    @JsonProperty
+    private String icpc_id;
+
+    @JsonProperty
+    private String name;
+
+    @JsonProperty
+    private String formal_name;
+
+    @JsonProperty
+    private String country;
+
+    @JsonProperty
+    private CLICSFileReference [] country_flag;
+
+    @JsonProperty
+    private String url;
+
+    @JsonProperty
+    private String twitter_hashtag;
+
+    @JsonProperty
+    private String twitter_account;
+
+    @JsonProperty
+    private CLICSLocation location;
+
+    @JsonProperty
+    private CLICSFileReference [] logo;
+
+    public CLICSOrganization() {
+        // for Jackson deserializer
+    }
+
+    /**
+     * Fill in organization information properties
+     * TODO: fix to use organization obtained from organizations.json.  From insitutions.tsv, we get this:
+     * [0]:1329 [1]:New York University [2]:NYU
+     * but only need [1].  We get the rest from the a typical account.
+     *
+     * @param account typical account using this organization
+     * @param orgFields String array of fields that correspond to institutions.tsv.  Gack!
+     */
+    public CLICSOrganization(String orgId, Account account, String [] orgFields) {
+        id = orgId;
+        icpc_id = orgId;
```

Comment:
It seems weird that `orgId` is assigned to both these fields.  If we don't have any notion of "organization" within PC2, then perhaps one of these should be NULL instead of set to the value of the other?  (I'm not sure where this gets used, so I'm a bit stabbing in the dark; it just seems weird to me to assign the same thing to two different fields when those two fields clearly exist for some separate reasons...)

Reply:
In all cases that I can think of, the `id` and `icpc_id `are the same, at least for DOMjudge and PC2.   The id property can be anything a CCS wants.  We (PC2) just happen to say we're going to use the **CMS ID** (which is the `icpc_id`) for the `id`.  Perfectly legal.